README.md

```diff
@@ -134,6 +134,42 @@ const sampleNonNullable: ComplexObjectNonNullable = {
     array: [{bar: null}], // Error: Type 'null' is not assignable to type 'number'
   }
 }
+
+type SimpleType = {
```

Comment:
please update title of this readme section to add info about this type

Reply:
@krzkaczor done